llvm/test/tools/llvm-objdump/XCOFF/private-headers-option.test

```diff
@@ -0,0 +1,38 @@
+## Test the --private-headers option for XCOFF object files.
+
+# RUN: yaml2obj -DMAGIC=0x1DF %s -o %t_xcoff32.o
+# RUN: yaml2obj -DMAGIC=0x1F7 %s -o %t_xcoff64.o
+# RUN: llvm-objdump --private-headers %t_xcoff32.o |\
+# RUN:   FileCheck %s --check-prefixes=CHECK32 --match-full-lines
```

Comment:
I'm guessing the formatting is important, in which case you should also have `--strict-whitespace`. You may also wish to consider `--implicit-check-not={{.}}`, to ensure that no output exists except that which is checked for.

Reply:
I do check not all output of `llvm-objdump --private-headers`, for example:
`private-headers-option.test.tmp_xcoff32.o:        file format aixcoff-rs6000` , so I can not use `--implicit-check-not={{.}}` here